docs/content/en/main/concepts/investigations/edge-multi-cluster.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Edge multi-cluster"
+linkTitle: "Edge multi-cluster"
+weight: 1
+description: >
+  Investigation of edge multi-cluster
+---
+
+Kubernetes offers a widely adopted, state-based management platform to declaratively develop and operate containerized workloads. Kubernetes allows a 
+developer to create deployments that are node-agnostic and can be constrained to operate in places where certain conditions are met. Developers and 
+Operators understand that pods will be rescheduled to execute on nodes wherever constraints allow them to run. This approach has satisfied the large
+   majority of use cases in the cloud native community.
+
+There is a growing community of cloud native developers and operators that require more fine-grained control
+ over the distribution of pods due to the nature of their business. Telecommunications, industrial manufacturers, and automotive
+  companies would like to rely on Kubernetes state-based management but they need more flexibility and specificity to accomodate for requirements like
+  large numbers of edge locations, disconnected and autonomous operation, and constrained cluster size. In the edge multi-cluster usecase, there are
```

Comment:
I'm not sure that the "fine-grained control" and "need more flexibility" is the incentive for Edge-mc. I would suggest to replace the whole paragraph above and  simply go with something like:  "...  would like to rely on Kubernetes state-based management but have unique requirements that are not covered by Kubernets or KCP, like....